README.md

```diff
@@ -23,6 +24,24 @@ fn main() {
 }
 ```
 
+**NOTE:** The computed width values may not match the actual rendered column
+width. For example, the woman scientist emoji comprises of a woman emoji, a
+zero-width joiner and a microscope emoji.
+
+```rust
+extern crate unicode_width;
+use unicode_width::UnicodeWidthStr::width;
```

Comment:
This code doesn't compile, you can't import non-instance methods.

Reply:
Yeah, I can't get the rustdoc test to work locally for some reason, compiles now.